opencti-platform/opencti-front/src/private/components/entities/events/EventKnowledge.js

```diff
@@ -63,6 +63,28 @@ class EventKnowledgeComponent extends Component {
               />
             )}
           />
+          <Route
+            exact
+            path="/dashboard/entities/events/:eventId/knowledge/targets"
+            render={(routeProps) => (
+              <EntityStixCoreRelationships
+                entityId={event.id}
+                relationshipTypes={['targets']}
+                isRelationReversed
+                entityLink={link}
+                allDirections={true}
+                stixCoreObjectTypes={[
```

Comment:
According to the stix.ts file, the campaign entity is not available for an event

Reply:
Maybe it's better to don't pass any entities ?

Reply:
Fixed

Reply:
The list still not aligned with the stix.ts file. Maybe some relationship is missing in this file.

Existing
ENTITY_TYPE_INCIDENT
ENTITY_TYPE_INTRUSION_SET
ENTITY_TYPE_MALWARE
ENTITY_TYPE_THREAT_ACTOR
ENTITY_TYPE_TOOL

Your adding
PR ENTITY_TYPE_CAMPAIGN

Reply:
Fixed